packages/api-explorer/src/form-components/MultiSchemaField.jsx

```diff
@@ -0,0 +1,313 @@
+const React = require('react');
+const BaseMultiSchemaField = require('@readme/oas-form/src/components/fields/MultiSchemaField').default;
+const fieldProps = require('@readme/oas-form/src/types').fieldProps;
+const { getDefaultFormState, deepEquals } = require('@readme/oas-form/src/utils');
+
+const Component = React.Component;
+/**
+ * Find the property that the discriminator is referencing.
+ */
+function findDiscriminatorField(schema, options, property) {
+  if (schema.properties?.[property]) {
+    return schema.properties[property];
+  }
+
+  // I need to return early, so I can't .forEach, wish I could use a for-of loop!
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < options.length; i++) {
+    const option = options[i];
+    if (option.properties?.[property]) {
```

Comment:
We'll need to load https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining in order for us to safely use this. You should be able to add it as a devDep in the root directory and then add it into the `plugins` array in `.babelrc`.

Reply:
Made the change locally, how would I ensure it's working? Optional chaining was working for me without this plugin

Reply:
You should be able to run `npm run build` and then look at the source to make sure that `?.` isn't present.

Reply:
Talked offline and the optional chaining plugin is now just part of `@babel/preset-env`.

Reply:
👍  change pushed up